### DISCUSSION TITLE
Link color options for the future?

### DISCUSSION THREAD

**Discussion by ShaughnBurnison** [jendrikseipp/rednotebook#678] [Ideas]
An option to change the link color would be useful. Running on cinnamon desktop using dark mode, links in the "Edit" mode are light blue and easy to read. But in "Preview" mode change to a very dark blue that is nearly impossible to read.  

![edit_mode_link](https://user-images.githubusercontent.com/42915033/221994329-1e663e3d-b15f-4951-ad92-325cbc96a1e6.png)
![preview_mode_link](https://user-images.githubusercontent.com/42915033/221994333-2d070c1b-c6d9-4dab-9a78-364b04b0f151.png)

Great app. Thanks,

**Comment by jendrikseipp** [comment]
Good catch! Instead of making this an option, I think it's best to simply choose sensible default colors. Care to make a PR? The code is in rednoetbook/util/markup.py.
